app/models/manageiq/providers/ansible_tower/shared/inventory/parser/configuration_script_source.rb

```diff
@@ -0,0 +1,8 @@
+module ManageIQ::Providers::AnsibleTower::Shared::Inventory::Parser::ConfigurationScriptSource
+  include parent::AutomationManager
```

Comment:
I think you can even use just one parser. All can be driven by Collector providing only ConfigurationScriptSource data